drivers/python/age/__init__.py

```diff
@@ -24,12 +24,13 @@ def version():
     return VERSION.VERSION
 
 
-def connect(dsn=None, graph=None, connection_factory=None, cursor_factory=ClientCursor, **kwargs):
+def connect(dsn=None, graph=None, connection_factory=None, cursor_factory=ClientCursor, superuser:bool=True, **kwargs):

```

Comment:
To be consistent with rest of the parameters, I would avoid specifiying the type.

Reply:
Makes sense.